trade_remedies_caseworker/templates/v2/merge_organisations/merge_complete.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'v2/govuk/base.html' %}
+
+{% block page_title %}Complete{% endblock page_title %}
+
+{% block back_button %}{% endblock back_button %}
+
+{% block row_content %}
+
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1 class="govuk-panel__title">
+                Matching organisations resolved
```

Comment:
Mismatch here as well.

```suggestion
{% block page_title %}Matching organisations resolved{% endblock page_title %}

{% block back_button %}{% endblock back_button %}

{% block row_content %}

    <div class="govuk-grid-column-two-thirds">
        <div class="govuk-panel govuk-panel--confirmation">
            <h1 class="govuk-panel__title">
                Matching organisations resolved
```